Merge Grid Implementation

AlexanderSelmanovic opened the pull request.

AlexanderSelmanovic committed:
Object movable in grid system

AlexanderSelmanovic committed:
Tilemap updates

AlexanderSelmanovic committed:
Tilemaps can now be created programatically

AlexanderSelmanovic committed:
Tilemap changed to Rectangular

Tilemap changed to Rectangular, changed all assets from isometric to rectangular. All tiles now accessable in code

AlexanderSelmanovic committed:
Update SampleScene.unity

AlexanderSelmanovic committed:
Tiles halfway done

AlexanderSelmanovic committed:
Finished, but missing one type of asset

AlexanderSelmanovic committed:
Horizontal Assets added

AlexanderSelmanovic committed:
Grid finished

AlexanderSelmanovic committed:
Created Prefab for the GameZone

AlexanderSelmanovic committed:
Update from master

AlexanderSelmanovic committed:
Removed unused objects

AlexanderSelmanovic committed:
Added packages

AlexanderSelmanovic committed:
Merge branch 'master' into New-Unity-Project

AlexanderSelmanovic committed:
Removed starting Tiles for Gamezone

AlexanderSelmanovic committed:
Update GameZone, walkable tiles added

gimme reviewed (changes requested):
- The scripts "GameZone.cs" and "TilesResourceLoader.cs" should be put into a subfolder (e.g., "Assets/Scripts/MySubfolder/").
- The scripts should also preferably be put into a namespace (e.g., "Ecosystem/MySubfolder").
- Is the "Collider Debug" layer intended to stay?

AlexanderSelmanovic committed:
Scenary added

Namespace created for Grid
Scenary assets added
Scenary logic added

AlexanderSelmanovic committed:
Different kinds of scenary added

AlexanderSelmanovic committed:
Scenary finished

Scenary and tileassets moved to separate classes.
Unused scenary removed.
Grid prefab moved to support models

AlexanderSelmanovic committed:
List of scenary now added to the prefab

AlexanderSelmanovic committed:
Changed range for randomizer
